### ISSUE TITLE
Displays mismatched modlist 

### ISSUE THREAD

**Issue by youngchan01** [ichttt/MoreSoundConfig#8]
Displays mismatched modlist with server when this mod is client only

**Comment by XFactHD** [comment]
@ichttt You are missing the `DISPLAYTEST` extension point in the mod constructor:
```
ModLoadingContext.get().registerExtensionPoint(
    ExtensionPoint.DISPLAYTEST,
    () -> Pair.of(() -> FMLNetworkConstants.IGNORESERVERONLY, (a, b) -> true)
);
```

**Comment by ichttt** [comment]
God dammit, right

**Comment by ichttt** [comment]
Fixed in the latest releases for 1.16.5 and 1.17.1
